src/custom/theme/baseTheme.tsx

```diff
@@ -72,18 +73,27 @@ export function themeVariables(darkMode: boolean, colorsTheme: Colors) {
     header: {
       border: `1px solid ${colorsTheme.border}`
     },
+    buttonSizes: {
+      [ButtonSize.BIG]: css`
+        font-size: 26px;
+      `,
+      [ButtonSize.MEDIUM]: css`
```

Comment:
Small nit pick: Would call the MEDIUM -> DEFAULT one. Easier to understand when to use it.

Reply:
i find it a bit strange to call it "default", since i'm enumerating the sizes, I just made MEDIUM the default for convenience. Anyways, if its easier, I change it

Reply:
As said a nit pick, but I'd envision that when using the sizes in practice, it's good to know (also for someone not directly in the know of all the available sizes) that 'default' is the standard size (you don't have to think what MEDIUM is in the context of all sizes). I'd argue it's semantically consistent with how we use native CSS properties like `font-weight: normal` (we could even call it `normal` instead of `default`).